doc/manual-install.md

```diff
@@ -236,19 +236,12 @@ invoke-rc.d nginx restart
 ```
 
 ## Running behind NAT
-Jitsi-Videobridge can run behind a NAT, provided that all required ports are routed (forwarded) to the machine that it runs on. By default these ports are (TCP/443 or TCP/4443 and UDP 10000-20000).
+Jitsi-Videobridge can run behind a NAT, provided that all required ports are routed (forwarded) to the machine that it runs on. By default these ports are (TCP/443 or TCP/4443 and UDP 10000).
 
-The following extra lines need to be added the file `~/.sip-communicator/sip-communicator.properties` (in the home directory of the user running the videobridge):
+The following extra lines need to be added the file `/etc/jitsi/videobridge/sip-communicator.properties`
```

Comment:
This is not correct, as you can see in the install instructions the folder is `~/.sip-communicator`. To use /etc/jitsi/videobridge/ you need extra params which are not described in the doc.

Reply:
According to https://github.com/jitsi/jitsi-meet/blob/d7eea8abbcd76e956128bfa137a943031928029d/doc/quick-install.md#advanced-configuration it is correct?